application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadActivityUpdater.java

```diff
@@ -71,33 +63,18 @@ private void updateActivityForGuild(Guild guild) {
     }
 
     private void updateActivityForThread(ThreadChannel threadChannel) {
-        determineActivity(threadChannel)
-            .flatMap(threadActivity -> helper.changeChannelActivity(threadChannel, threadActivity))
-            .queue();
+        helper.changeChannelActivity(threadChannel, determineActivity(threadChannel)).queue();
     }
 
-    private static RestAction<HelpSystemHelper.ThreadActivity> determineActivity(
-            MessageChannel channel) {
-        return channel.getHistory().retrievePast(ACTIVITY_DETERMINE_MESSAGE_LIMIT).map(messages -> {
-            if (messages.size() >= ACTIVITY_DETERMINE_MESSAGE_LIMIT) {
-                // There are likely even more messages, but we hit the limit
-                return HelpSystemHelper.ThreadActivity.HIGH;
-            }
-
-            Map<User, List<Message>> authorToMessages = messages.stream()
-                .filter(Predicate.not(HelpThreadActivityUpdater::isBotMessage))
-                .collect(Collectors.groupingBy(Message::getAuthor));
-
-            boolean isThereActivity = authorToMessages.size() >= 2 && authorToMessages.values()
-                .stream()
-                .anyMatch(messagesByAuthor -> messagesByAuthor.size() >= 2);
-
-            return isThereActivity ? HelpSystemHelper.ThreadActivity.MEDIUM
-                    : HelpSystemHelper.ThreadActivity.LOW;
-        });
-    }
+    private static HelpSystemHelper.ThreadActivity determineActivity(ThreadChannel threadChannel) {
+        if (threadChannel.getMemberCount() < 3) {
+            return HelpSystemHelper.ThreadActivity.LOW;
+        }
+
+        if (threadChannel.getMessageCount() > 15) {
+            return HelpSystemHelper.ThreadActivity.HIGH;
+        }
 
-    private static boolean isBotMessage(Message message) {
-        return message.getAuthor().equals(message.getJDA().getSelfUser());
+        return HelpSystemHelper.ThreadActivity.MEDIUM;
```

Comment:
Interesting. How does this behave with threads that are not "cached"? For example if you take the bot offline, then create a thread and start the bot afterwards. Will it be able to pick the correct numbers up, without queuing an action?

(@Tais993)

Reply:
> For example if you take the bot offline, then create a thread and start the bot afterwards
i tested that and it worked

> Will it be able to pick the correct numbers up, without queuing an action?
no idea

Reply:
okay. ill wait for a confirmation by @Tais993 that this approach works. then im good with it :)

Reply:
Iirc it works, message count is included by Discord

Reply:
it works i tested, but ig what zabu wants to ask is "without queuing an acting?" whatever that means

Reply:
im just surprised this method comes along without a `RestAction`. so i suppose it works with the cache. but i could imagine cases where the cache doesnt know the thread yet. so its important to understand whether it would work in these cases (@Tais993 ?)

Reply:
i mean u r sending the thread object so it should work without any issue

Reply:
If the cache doesn't know about the thread, this wouldn't run to begin with, and that has always been the same.
So no this shouldn't cause issues, to make sure, we can test it in the develop server.

But from my knowledge, this shouldn't cause any new issues we didn't have before.

Reply:
if u say so. to me that isnt necessarily super obvious. just because we have a cached thread, doesnt necessarily mean that it also knows about all its message history. hence why im careful.
but alright

Reply:
Well, it doesn't know about the history, it only knows that it contains ... Messages (if above 50, it gives 50)